ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh

```diff
@@ -456,7 +483,7 @@ if [[ "$CNF_BRANCH" == *"4.17"* ]]; then
     create_tests_temp_skip_list_17
     export GINKGO_PARAMS=" --timeout 230m -slow-spec-threshold=0.001s -v --show-node-events --json-report test_ginkgo.json --flake-attempts 4"
 fi
-if [[ "$CNF_BRANCH" == *"4.18"* ]] || [[ "$CNF_BRANCH" == *"master"* ]]; then
+if [[ "$CNF_BRANCH" == *"4.18"* ]] || [[ "$CNF_BRANCH" == *"4.19"* ]] || [[ "$CNF_BRANCH" == *"master"* ]]; then
```

Comment:
We have created 4.19 skip list function above but we are not using that function, I think it will be better to move 4.18 to separate condition and for 4.19 and master use 4.19 skiplist function.